index.html

```diff
@@ -78,6 +78,17 @@ <h2>Add New Recipe</h2>
 
     <div id="recipe-container"></div>
 
+    <div id="image-container"></div>
+
+    <div id="recipe-container"></div>
+
+    <input type="text" id="searchInput" placeholder="Search by title" />
+    <button onclick="searchRecipe()">Search</button>
```

Comment:
As mentioned [in the last PR](https://github.com/antuanettaV/ProjectRecipes/pull/1#discussion_r1667664690), prefer adding events with `addEventListener`.